src/main/plugins/org.dita.pdf2/xsl/fo/links.xsl

```diff
@@ -407,9 +406,7 @@ See the accompanying license.txt file for applicable licenses.
           <fo:list-item xsl:use-attribute-sets="related-links.ul.li">
             <fo:list-item-label xsl:use-attribute-sets="related-links.ul.li__label">
               <fo:block xsl:use-attribute-sets="related-links.ul.li__label__content">
-                <fo:inline>
```

Comment:
Returning to this so I can bring it up to date and close it out.

I don't see any reason we generate `fo:inline` in at least some of these case. I originally left those alone because I wanted to make the least invasive change I could. Now testing and moving to the appropriate containing block where that makes sense.